Games/DO TA/import random.py

```diff
@@ -0,0 +1,56 @@
+import random
+
+random_num=[1, 2, 3, 4, 5, 6 , 7 , 8, 9, 10, 11 ,12 , 13, 14, 15, -1, -2, -3, -4, -5, -6 , -7 , -8, -9, -10]
```

Comment:
You can control the range, etc. directly with the `random` library`, without explicitly listing all the possible values.
https://docs.python.org/3/library/random.html